gitgud/skills/newbasics/__init__.py

```diff
@@ -0,0 +1,44 @@
+from gitgud.skills.level_builder import BasicLevel
+from gitgud.skills.util import Skill
+from gitgud import operations
+from gitgud.skills.user_messages import firstcommit_status
+
+
+class FirstCommit(BasicLevel):
+    def _setup(self):
+        file_operator = operations.get_operator()
+        file_operator.destroy_repo()
+        file_operator.use_repo()
+
+    def status(self):
+        firstcommit_status(*self._get_state())
+
+    def _get_state(self):
```

Comment:
Since it would be totally fine for us to call this from another part of the code, I think we can call this method `get_state` without the underscore. @sahansk2 Do you think this conflicts with our other `_test`/`_setup` naming scheme?